.gitattributes

```diff
@@ -1 +1,2 @@
 .githubmap merge=union
+PendingReleaseNotes merge=union
```

Comment:
another way to address this is to add new entries in random places instead of adding them at the end of this file. but yeah, it's simpler to leave this to git.